src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexGeneratorOutputTests.cs

```diff
@@ -659,10 +727,10 @@ private bool TryMatchAtCurrentPosition(ReadOnlySpan<char> inputSpan)
 
                                                 // Match a character in the set [^>\s] atomically at least once.
                                                 {
-                                                    int iteration3 = 0;
-                                                    while ((uint)iteration3 < (uint)slice.Length && ((ch = slice[iteration3]) < 128 ? ("쇿\uffff\ufffe뿿\uffff\uffff\uffff\uffff"[ch >> 4] & (1 << (ch & 0xF))) != 0 : Utilities.Base.CharInClass((char)ch, "\u0001\u0002\u0001>?d")))
+                                                    int iteration3 = slice.IndexOfAnyExcept92ADA2AB68AD19F57A1E78FA973AF4ED3FB64C5AB5DF24C6A8EE0B8823EC27CF();
```

Comment:
While this doesn't look very nice, I don't have a good suggestion for what could we do to have a better name